src/plots/plot.ts

```diff
@@ -388,6 +388,7 @@ export class Plot extends Component {
    * Updates the extents associated with each attribute, then autodomains all scales the Plot uses.
    */
   protected _updateExtents() {
+    this._resetEntityStore();
```

Comment:
why is this needed here now? isn't all we need done through the `scale.onUpdate`?